### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ArslanYM**
Added Link to the original React Play repo
rp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # docsite
 Source Code of the ReactPlay Documentation Website
-
+# Link to the original repo of ReactPlay:
```
Comment (Sachin-chaurasiya) [inline README.md:3]: Hello @ArslanYM , thanks for the PR,

Can we just put simple text link instead of putting the heading like this?

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ArslanYM) [reply README.md:3]: yeah that seems more sophisticated

[the remark this answers is not in the document]
Comment (ArslanYM) [reply README.md:3]: Shall I make the changes and add a pr ?

[the remark this answers is not in the document]
Comment (Sachin-chaurasiya) [reply README.md:3]: @ArslanYM , We already have the original repo link in the README, can you check?

[the remark this answers is not in the document]
Comment (ArslanYM) [reply README.md:3]: yeah but its not propely made :

d

[the remark this answers is not in the document]
Comment (Sachin-chaurasiya) [reply README.md:3]: Yeah, it should be a link, you can fix that.

[the remark this answers is not in the document]
Comment (ArslanYM) [reply README.md:3]: on it

[the remark this answers is not in the document]
Comment (ArslanYM) [reply README.md:3]: Works?

new

[the remark this answers is not in the document]
Comment (ArslanYM) [reply README.md:3]: @Sachin-chaurasiya

[the remark this answers is not in the document]
Comment (ArslanYM) [reply README.md:3]: PR #37  fixes the issue

**Comment by ArslanYM** [comment]
Resolved in #37

**Pull request closed without merging**